tasks_management/signals.py

```diff
@@ -0,0 +1,70 @@
+from typing import Dict
+
+from core.service_signals import ServiceSignalBindType
+from core.signals import bind_service_signal
+from tasks_management.models import TaskGroup, Task
+
+
+def resolve_task_any(task, business_status: Dict[str, str]):
+    if any(status == "APPROVED" for status in business_status.values()):
```

Comment:
Why you don't use here constant value  like in Task.Status.FAILED? It should be Task.Status.APPROVED.

Reply:
It is not a task status but rather businessStatus. However I defined it in new file "utils" and use it from there.